code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -372,12 +372,21 @@
 		new /obj/item/weapon/reagent_containers/syringe/ld50_syringe/choral(src)
 
 
+// These are special snowflakes that need to be in a global list.
 /obj/structure/closet/secure_closet/brig
 	name = "brig locker"
 	req_access = list(access_brig)
 	anchored = 1
 	var/id = null
 
+	Initialize()
+		. = ..()
+		brig_closets += src
+
+	Destroy()
+		brig_closets -= src
+		return ..

```

Comment:
How did this compile? `..()`.

Turns out, `..` is **also** a valid symbol. Thanks BYOND!